spark/hive-site.xml

```diff
@@ -0,0 +1,63 @@
+<configuration>
```

Comment:
@akuksin Since everything will be run from the cluster(even if you proxy to local) we should update hostnames